flytekit/core/promise.py

```diff
@@ -811,6 +811,8 @@ def extract_obj_name(name: str) -> str:
 def create_and_link_node_from_remote(
     ctx: FlyteContext,
     entity: HasFlyteInterface,
+    _inputs_not_allowed: typing.Set[str] = None,
+    _ignorable_inputs: typing.Set[str] = None,
```

Comment:
do we really need to verify this here? Can you an example (maybe a unit test?) where this breaks, so that it becomes clear why we need it?

Reply:
done. good idea to add unit test and also some nice docs